inc/environment_indicator/namespace.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Altis\Cloud\Environment_Indicator;
+
+use function Altis\get_environment_name;
+use function Altis\get_environment_type;
+
+function bootstrap() {
+	add_action( 'wp_before_admin_bar_render', __NAMESPACE__ . '\\add_admin_bar_env_info' );
+}
+
+/**
+ * Add menu item to admin bar to display environment info:
+ * - environment type
+ * - environment name
+ * - URL to Altis dashboard for the environment
+ */
+function add_admin_bar_env_info() {
+	if ( ! is_admin_bar_showing() ) {
+		return;
+	}
+
+	// Specify capabilities for which environment indicator is shown in the admin bar.
+	$capabilities[] = 'manage_options';
+
+	if ( is_multisite() ) {
+		$capabilities[] = 'manage_network_options';
+	}
+
+	$capabilities = apply_filters( 'altis_env_indicator_capabilities', $capabilities );
+	foreach ( $capabilities as $cap ) {
+		if ( ! current_user_can( $cap ) ) {
+			return;
+		}
+	}
+
+	global $wp_admin_bar;
+	// Specify admin bar menu item info and styling.
+	$indicator_colour = 'red';
+	$indicator_colour = apply_filters( 'altis_env_indicator_colour', $indicator_colour, get_environment_type() );
+
+	$indicator_text = strtoupper( get_environment_type() );
+	$indicator_text = apply_filters( 'altis_env_indicator_text', $indicator_text, get_environment_type() );
+
+	$bar_item_style = '<style>
+#wpadminbar #wp-admin-bar-altis-env-indicator>.ab-item:before {
+	content: "\f547";
+}
+#wpadminbar #wp-admin-bar-altis-env-indicator,
+#wpadminbar #wp-admin-bar-altis-env-indicator>.ab-item:hover,
+#wpadminbar #wp-admin-bar-altis-env-indicator>.ab-item:focus,
+#wpadminbar #wp-admin-bar-altis-env-indicator>.ab-item:visited {
+	background: ' . $indicator_colour . ' ! important;
```

Comment:
I think we should control the indicator colours ourselves in a CSS file, we have an assets folder in the root of this package. See the branding namespace for an example.

Using the different class names we can set the colour. I'd like to avoid using `!important` if possible too.

Reply:
Colours based on our branding guidelines https://drive.google.com/drive/folders/1GnnbPJ87pnnjbklts656byxkbfutr1J6:

- local: `#5AC5B1` Green
- development: `#06A9CE` Aqua
- staging: `#F8DA6D` Yellow
- production: `#ED7B9D` Pink

Make sure the link text is visible, the dark blue `#152A4E` should be suitable.